### ISSUE TITLE
That interactive mode...

### ISSUE THREAD

**Issue by Simn** [HaxeFoundation/hxcpp#158]
```
Before this can be used, you need to:
 1. Rebuild the main command-line tool, this can be done with:
     cd tools/hxcpp
     haxe compile.hxml
 2. Build the binaries appropriate to your system(s), this can be done with:
     cd project
     neko build.n

Would you like to do this now [y/n]
```

This is rather annoying.

**Comment by hughsando** [comment]
Is this coming up when you have a correctly build hxcpp (bug), or when you
don't, but you also do not want to see the message?

You can set the environment or define HXCPP_NONINTERACTIVE and never see
this message again - we should probably do this in the automated tests.

**Comment by Simn** [comment]
I just updated hxcpp yesterday and one of my tests which invoked haxe through the haxe process API got stuck without any message. I only figured out what happened after invoking it from command line.

In general I don't really like if command lines talk back to me by default. However, shortly after this I came across the same problem while using git bisect, so I suppose it's not uncommon. I don't really understand why it's necessary to ask the user here, but now that I know about it it's fine with me.
